src/js/background.js

```diff
@@ -762,7 +768,6 @@ Badger.prototype = {
     }
 
     chrome.browserAction.setIcon({tabId: tab.id, path: iconFilename});
-    chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
```

Comment:
Nice, now this is localized properly by the `"default_title"` key in the manifest